test/test.cpp

```diff
@@ -17,14 +17,15 @@ using namespace tiny_dnn::activation;
 // FIXME(Randl) switching previous and next lines breaks tests
 #include "test_average_pooling_layer.h"
 // TODO(yida): fix broken test
-//#include "test_average_unpooling_layer.h"
+#include "test_average_unpooling_layer.h"
 #include "test_batch_norm_layer.h"
 #include "test_concat_layer.h"
 #include "test_convolutional_layer.h"
 #include "test_core.h"
 #include "test_deconvolutional_layer.h"
 #include "test_dropout_layer.h"
 #include "test_fully_connected_layer.h"
```

Comment:
Probably should remove this. Conflicting test names and no point in testing old code.